funcx_sdk/funcx/sdk/asynchronous/ws_polling_task.py

```diff
@@ -20,9 +20,9 @@ class WebSocketPollingTask:
 
     def __init__(self, funcx_client,
                  loop: AbstractEventLoop,
-                 atomic_controller,
+                 atomic_controller=None,
```

Comment:
Maybe add a line to the docstring below that the atomic_controller is required when called from a non async environment.